mt940/models.py

```diff
@@ -37,6 +37,38 @@ def __new__(cls, *args, **kwargs):
             # No need to do anything special there :)
             return datetime.date.__new__(cls, *args, **kwargs)
 
+class DateTime(datetime.datetime, Model):
```

Comment:
could we merge this with the `Date` class? and return a datetime if hour, minute etc. is given?